### PR TITLE
feature(resolver): multiresolution new features

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-1114](https://emurgo.atlassian.net/browse/YOMO-1114)

[YOMO-1114]: https://emurgo.atlassian.net/browse/YOMO-1114?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx [hunk 1]
```diff
@@ -43,35 +45,37 @@ export const SelectNameServer = () => {
...
 
-          <Spacer height={16} />
+          {!receiver.selectedNameServer && shouldShow && (
+            <>
+              <Spacer height={16} />
 
-          <ButtonGroup initial={initial} labels={labels} onSelect={handleOnSelectNameServer} />
-
-          <Spacer height={16} />
+              <ShowManyAddressWarning />
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx:60]: just confirming the warning now is displayed below the options?

Comment (banklesss) [reply apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx:60]: yes it is! https://www.figma.com/file/KeBVo6rWNYFLXwTphz9lHi/Mobile-App?node-id=14403%3A47040&mode=dev

File: apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx [hunk 2]
```diff
@@ -90,9 +94,24 @@ const styles = StyleSheet.create({
...
   },
 })
+
+const bold = {b: (text) => <Text bold>{text}</Text>}
+
+export const useLocalStrings = () => {
+  const intl = useIntl()
+
+  return {
+    manyNameServersWarning: intl.formatMessage(messages.manyNameServersWarning, bold),
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx]: add to the `useStrings` of the funnel, and return as a function that takes the a react node.

Comment (banklesss) [reply apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx]: done

**Pull request merged**